File: Task 2/anupm12/regex.cpp
#include<iostream>
#include<regex>
using namespace std;

int main()
{
    string email,password;

    cout<<"Enter your email"<<endl;
    cin>>email;
    cout<<"Enter your password"<<endl;
    cin>>password;

    regex remail("(.*)(@gmail.com)");
    regex rpassword("(.*){8}");

    if(regex_match(email,remail))
    {
        if(regex_match(password,rpassword))
            cout<<"Login successful"<<endl;
        else
            cout<<"You entered incorrect password"<<endl;
    }
    else
        cout<<"Invalid email"<<endl;
}

File: Task 2/vishvajeet/regex.py
import re

userid = input("Enter id : ")
password = input("Enter password : ")

user = re.search("^[\w.+\-]+@gmail\.com$",userid)
key = re.search(".{8,}$",password)

if user:
    if key:
        print(" Login Successfully !")
    else:
        print("You Entered wrong password")

else:
    print("Email is invaild")
